src/ReactiveUI.Tests/Platforms/xaml/ActivationForViewFetcherTest.cs

```diff
@@ -95,14 +98,14 @@ public void IsHitTestVisibleDeactivatesFrameworkElement()
             new[] { true, false }.AssertAreEqual(activated);
         }
 
-        [Fact]
+        [WpfFact]
         public void FrameworkElementIsActivatedAndDeactivatedWithHitTest()
         {
             var uc = new TestUserControl();
             var activation = new ActivationForViewFetcher();
 
             var obs = activation.GetActivationForView(uc);
-            var activated = obs.CreateCollection();
+            var activated = obs.CreateCollection(scheduler: ImmediateScheduler.Instance);
```

Comment:
Yep, this is the exact problem I ran into with the `SchedulerShouldBeCurrentThreadInTestRunner` test under NCrunch. I dream of a day we can completely remove ambient contexts from ReactiveUI.